packages/create-webstone-app/template/plugin/structure/.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I love that we include this by default. It's such a smooth experience and shows how Webstone cares about the overall developer experience, not just about writing code.